.circleci/config.yml

```diff
@@ -203,11 +203,11 @@ run_tests: &run_tests
           sleep 20 # wait 20 seconds for file sync to finish
 
           if [[ $CIRCLE_BRANCH = "master" ]]; then
-            terminus remote:drush portlandor.dev -- cim -y
+            terminus remote:drush portlandor.dev -- csim config_dev -y
```

Comment:
Switch this back. `git checkout master .circleci/config.yml` to reset it back to master.